src/canvas.cpp

```diff
@@ -76,7 +83,9 @@ bool canvas::print_pdf(const char *file_name)
   cairo_paint(context);
 
   using namespace std::placeholders;
-  renderer g(context, std::bind(&camera::world_to_screen, m_camera, _1), &m_camera, surface);
+  camera pdf_cam({{0,0},{(double)surface_width, (double)surface_height}});
```

Comment:
Hi Tina,
I found that initializing the pdf_cam with m_camera is better than initializing them with the surface width and height (it makes the width and height of the pdf independent on the user world coordinates).
Can you make that change by modifying this line to 
`camera pdf_cam = m_camera;`
This also applies to other printing functions.

Reply:
sure